HW_5/task_1.py

```diff
@@ -0,0 +1,9 @@
+"""
+Напишите программу, удаляющую из текста все слова, в которых присутствуют буквы «а», «б» и «в».
+
+Ввод: значение типа <str>
+Вывод: значение типа <str>
+"""
+s = input()
+
+print(' ' .join(list(filter(lambda str_: not all(i in str_.lower() for i in 'абв'), s.split()))))
```

Comment:
Функцию `list` можно использовать.
```suggestion
print(' ' .join(filter(lambda str_: not all(i in str_.lower() for i in 'абв'), s.split())))
```